models/articles.models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection")
+
+exports.fetchArticleById = exports.fetchArticle = async (article_id) => {
+		const article = await db.query(
+		`SELECT *
+		FROM articles
+		WHERE article_id = $1;`,
+		[article_id]
+		)
+
+		if (!article.rows.length) {
+			return { status: 404, msg: `Article not found for article_id: ${article_id}` }
```

Comment:
Should we be returning this object? I would recommend revisiting the custom errors section of the error-handling notes

Reply:
Hoping you just mean to return it as Promise.reject instead? Thats what I've done anyway!